From 3e8973062fa2b2c7df6d07ac98bfd4aef6bff5e3 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <perex@perex.cz>
Date: Mon, 6 Jul 2020 16:47:57 +0200
Subject: [PATCH 36/51] hda hdmi: add HDMI4,HDMI5,HDMI6 devices

- fix the duplicate index value in the comment section
- change priorities

Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/HDA-Intel/Hdmi.conf  | 42 +++++++++++++++++++++++++++++++++++++++---
 ucm2/codecs/hda/hdmi.conf |  2 +-
 2 files changed, 40 insertions(+), 4 deletions(-)

diff --git a/ucm2/HDA-Intel/Hdmi.conf b/ucm2/HDA-Intel/Hdmi.conf
index 8b1843b91e6b..c8b6e77a0299 100644
--- a/ucm2/HDA-Intel/Hdmi.conf
+++ b/ucm2/HDA-Intel/Hdmi.conf
@@ -6,7 +6,7 @@ If.hdmi1 {
 		Define {
 			HdmiNum 1
 			HdmiPCM 3
-			HdmiPrio 500
+			HdmiPrio 1100
 		}
 		Include.hdmi1.File "/codecs/hda/hdmi.conf"
 	}
@@ -18,7 +18,7 @@ If.hdmi2 {
 		Define {
 			HdmiNum 2
 			HdmiPCM 7
-			HdmiPrio 600
+			HdmiPrio 1200
 		}
 		Include.hdmi2.File "/codecs/hda/hdmi.conf"
 	}
@@ -30,8 +30,44 @@ If.hdmi3 {
 		Define {
 			HdmiNum 3
 			HdmiPCM 8
-			HdmiPrio 700
+			HdmiPrio 1300
 		}
 		Include.hdmi3.File "/codecs/hda/hdmi.conf"
 	}
 }
+
+If.hdmi4 {
+	Condition { Type String Empty "" }
+	True {
+		Define {
+			HdmiNum 4
+			HdmiPCM 9
+			HdmiPrio 1400
+		}
+		Include.hdmi4.File "/codecs/hda/hdmi.conf"
+	}
+}
+
+If.hdmi5 {
+	Condition { Type String Empty "" }
+	True {
+		Define {
+			HdmiNum 5
+			HdmiPCM 10
+			HdmiPrio 1500
+		}
+		Include.hdmi5.File "/codecs/hda/hdmi.conf"
+	}
+}
+
+If.hdmi6 {
+	Condition { Type String Empty "" }
+	True {
+		Define {
+			HdmiNum 6
+			HdmiPCM 11
+			HdmiPrio 1600
+		}
+		Include.hdmi5.File "/codecs/hda/hdmi.conf"
+	}
+}
diff --git a/ucm2/codecs/hda/hdmi.conf b/ucm2/codecs/hda/hdmi.conf
index c22f61521756..ba80fefc214e 100644
--- a/ucm2/codecs/hda/hdmi.conf
+++ b/ucm2/codecs/hda/hdmi.conf
@@ -5,7 +5,7 @@ If.hdmi {
 	}
 	True {
 		SectionDevice."HDMI${var:HdmiNum}" {
-			Comment "HDMI${var:HdmiNum}/${var:HdmiNum} Output"
+			Comment "HDMI${var:HdmiNum} Output"
 
 			EnableSequence [
 				cset "name='IEC958 Playback Switch' on"
-- 
2.16.4

